> e:\uni\invatare adanca\lab5\dataset.py(89)__init__()
-> self.X= cv2.cvtColor(self.X, cv2.COLOR_BGR2RGB)
(1729, 250, 250, 3)
> e:\uni\invatare adanca\lab5\dataset.py(90)__init__()
-> self.Y= cv2.cvtColor(self.Y, cv2.COLOR_BGR2RGB)
(1729, 250, 3)
*** SyntaxError: invalid syntax
Traceback (most recent call last):
  File "e:\uni\Invatare ADANCA\lab5\cod_curat.py", line 288, in <module>
    unet_training_loop()
  File "e:\uni\Invatare ADANCA\lab5\cod_curat.py", line 176, in unet_training_loop
    dataset = LFWDataset(base_folder='data', transforms=None, download=False)
  File "e:\uni\Invatare ADANCA\lab5\dataset.py", line 90, in __init__
    self.Y= self.Y[..., ::-1]
  File "e:\uni\Invatare ADANCA\lab5\dataset.py", line 90, in __init__
    self.Y= self.Y[..., ::-1]
  File "C:\Users\byefu\AppData\Local\Programs\Python\Python310\lib\bdb.py", line 90, in trace_dispatch
    return self.dispatch_line(frame)
  File "C:\Users\byefu\AppData\Local\Programs\Python\Python310\lib\bdb.py", line 115, in dispatch_line
    if self.quitting: raise BdbQuit
bdb.BdbQuit